docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.7"
+
+services:
+  localstack-s3:
+    image: localstack/localstack:latest
```

Comment:
this is a huge image, but I haven't found a better option with a liberal license

Reply:
do they have any good released version more stable than `latest`?

Reply:
Doesn't appear to be. `latest` is the latest version of the image.